packages/@aws-cdk/aws-ec2-alpha/lib/route.ts

```diff
@@ -402,6 +436,103 @@ export class NatGateway extends Resource implements IRouteTarget {
   }
 }
 
+/**
+ * Creates a peering connection between two VPCs
+ * @resource AWS::EC2::VPCPeeringConnection
+ */
+export class VPCPeeringConnection extends Resource implements IRouteTarget {
+
+  /**
+   * The type of router used in the route.
+   */
+  readonly routerType: RouterType;
+
+  /**
+   * The ID of the route target.
+   */
+  readonly routerTargetId: string;
```

Comment:
```suggestion
  public readonly routerTargetId: string;
```

Reply:
Keeping without public for consistency with other classes